pkg/cmd/pipelinerun/list_test.go

```diff
@@ -22,18 +22,76 @@ import (
 	"github.com/jonboulle/clockwork"
 	"github.com/knative/pkg/apis"
 	"github.com/spf13/cobra"
-	"github.com/tektoncd/cli/pkg/testutil"
+	tu "github.com/tektoncd/cli/pkg/testutil"
```

Comment:
This one is not needed anymore as https://github.com/tektoncd/pipeline/pull/854 is merged :angel:

Reply:
FYI, it won't work (for now ) as we have pinned down the version to the latest release of pipeline (0.3.1). Should master track pipeline/master instead and make `releases` in sync with pipeline?

Reply:
ah good point, I didn't thought of that :sweat_smile:. I think we can follow master and make releases in sync with pipeline indeed (or at least, when we release the cli, we pin pipeline)

Reply:
that sounds viable as well

Reply:
@hrishin would you be making that change in this PR?

Reply:
I could @sthaha, if we all agreed :+1:

Reply:
Let's do the dep/vendor bump in another PR :angel: